doc/source/development/contributing.rst

```diff
@@ -598,7 +598,7 @@ Building master branch documentation
 
 When pull requests are merged into the pandas ``master`` branch, the main parts of
 the documentation are also built by Travis-CI. These docs are then hosted `here
-<https://dev.pandas.io>`__, see also
+<https://pandas.pydata.org/pandas-docs/dev/>`__, see also
```

Comment:
```suggestion
<https://pandas.pydata.org/docs/dev/>`__, see also
```

(this shorter version also works)

Reply:
the shorter version does *not* seem to work (longer does)

Reply:
We use the shorter version in several places (eg the README, the issue template, etc). Do you have a link that doesn't work?